python/Produce_Config_File.py

```diff
@@ -86,24 +86,23 @@ def CyclicColor(DrawArg):
 	text_file.write("\t\t\tPlot_Name = '%s';\n"%(str(legEntry)))
 	text_file.write("\t\t\t[BEGIN_DATA]\n" )
 	text_file.write("\t\t\tVAR_INDEP,VAR_DEP,VAR_INDEP_ERR,VAR_DEP_ERR\n" )
-	DrawArg=DrawArg+1
 
 	for row in range(args.SelectRowStart,args.SelectRowEnd):
-		if bool(args.YaxisScale):
+		if args.YaxisScale=='true':
```

Comment:
This is fine, but keep in mind that this:

```
if args.YaxisScale:
```

Also works, since YaxisScale is boolean the if statement sees either 1 or 0 which evaluates to true or false.